README.md

```diff
@@ -29,7 +29,8 @@ make deploy
 
 ### Using the Operator
 
-1. Create the secrets that are to be used by the custom resource:
+1. Create a secrets.yaml that will store the secrets to be used by the custom resource:
+You may choose to create this file under /samples dir
```

Comment:
Can we remove this line since we're already mentioning `<path/to/secrets.yaml>` in the next instruction.